src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,92 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int LENGTH_MULTIPLIER = 2;
+    private static final float LOAD_FACTOR = 0.75f;
+    private Node<K, V>[] table;
+    private int size;
+
+    public MyHashMap() {
+        table = new Node[DEFAULT_CAPACITY];
+    }
 
     @Override
     public void put(K key, V value) {
-
+        isResize();
+        int index = getIndex(key);
+        if (table[index] != null) {
+            Node<K, V> node = table[index];
+            if (Objects.equals(node.key, key)) {
+                node.currentValue = value;
+                return;
+            }
+            while (node.next != null) {
+                node = node.next;
+                if (Objects.equals(node.key, key)) {
+                    node.currentValue = value;
+                    return;
+                }
+            }
```

Comment:
you can make this code little more simple:
```suggestion
            while (node.next != null) {
                if (Objects.equals(node.key, key)) {
                    node.currentValue = value;
                    return;
                }
                node = node.next;
            }
```